boost_manager/src/watcher.rs

```diff
@@ -34,7 +34,7 @@ where
         Box::pin(
             handle
                 .map_err(anyhow::Error::from)
-                .and_then(|result| async move { result.map_err(anyhow::Error::from) }),
+                .and_then(|result| async move { result }),
```

Comment:
huh? this doesn't actually do anything right?

Reply:
It does 
```  --> boost_manager/src/watcher.rs:34:9
   |
34 | /         Box::pin(
35 | |             handle.map_err(anyhow::Error::from),
36 | |             // .and_then(|result| async move { result }),
37 | |         )
   | |_________^ expected `Result<(), Error>`, found `Result<Result<(), Error>, Error>`
   |
   = note: expected enum `std::result::Result<(), _>`
              found enum `std::result::Result<std::result::Result<(), anyhow::Error>, _>````